src/ResizeSensor.d.ts

```diff
@@ -1,7 +1,12 @@
+export declare type ResizeSensorCallback = (size: { width: number; height: number; }) => void;
+
 declare class ResizeSensor {
-    constructor(element: (Element | Element[]), callback: Function);
-    detach(callback: Function): void;
-    reset(element: Element | Element[]): void;
+    constructor(element: Element | Element[], callback: ResizeSensorCallback);
+    detach(callback: ResizeSensorCallback): void;
+    reset(): void;
+
+    static detach(element: Element | Element[], callback: ResizeSensorCallback): void;
```

Comment:
callback is optional here, can you mark it as such please? :)

Reply:
Added the optional for detach callbacks.